stable/kong/Chart.yaml

```diff
@@ -10,5 +10,5 @@ maintainers:
 name: kong
 sources:
 - https://github.com/Kong/kong
-version: 0.3.0
+version: 0.3.1
```

Comment:
Though its not a breaking change, should we bump it to 0.4.0?

Reply:
Whatever you will. I'll correct this value if you wish.

Reply:
Yes, lets bump it 0.4.0